### ISSUE TITLE
[joist] test "Full Screen" feature

### ISSUE THREAD

**Issue by jonathanolson** [phetsims/tasks#24]
Testing candidates are at http://www.colorado.edu/physics/phet/dev/olsonjb/scenery-0.2-unoptimized-tests-4/.

Please pick a sim or two and test this feature out on all platforms. A summary of the expected behavior:

1. There is a "Full Screen" item under the PhET Menu (lower-right).
2. There is a check mark next to the "Full Screen" item when the sim is full-screen __as a result of pressing our "Full Screen" item__. Security restrictions seem to prevent us from detecting if the user presses F11 (or equivalent) to trigger full-screen.
3. Pressing it when there is NO check mark next to it will make the sim full-screen.
4. Pressing it when there is a check mark next to it will make the sim NOT full-screen.
5. Pressing "Escape" or equivalent keys when in sim-caused full-screen mode will both exit full-screen mode and remove the check-mark.

Basically, if anything with it seems broken when not using the browser's built-in menu or keys to trigger full-screen, let me know.

Additionally, if it causes really buggy behavior (can't get back to or out of full-screen) using a combination of the sim's "Full Screen" button and the browser's controls, let me know.

**Comment by pixelzoom** [comment]
@ariel-phet Since scenery/joist 0.2 has been moved to master, this feature is now enabled, and will be included in any published sims. So perhaps the priority of testing should be raised.

**Comment by pixelzoom** [comment]
I'll handle restoring the query parameter to make this optional, phetsims/joist#213.

**Comment by pixelzoom** [comment]
This feature is now off by default (see phetsims/joist#213). When testing is completed, please create an issue to make it "on" by default.

**Comment by pixelzoom** [comment]
To make the "Full Screen" menu item visible in the PhET menu, run a sim with "fullscreen" query parameter.

**Comment by pixelzoom** [comment]
5/28/15 dev meeting:  Have a developer test this to see if it's ready. If it's ready, publish a release candidate of some sim, then assign to a student tester.

**Comment by jonathanolson** [comment]
Will to a dev deployment of molecule-shapes with fullscreen enabled by default for testing (and will have a new sheet in the sim's test matrix).

**Comment by jonathanolson** [comment]
Added sheet to Molecule Shapes test matrix for this testing (https://docs.google.com/spreadsheets/d/1wfA1SVvOKK-d26ozNNeGDuuq5OtUpInGcEkfK_Eqvgg), and deployed Molecule Shapes 1.0.1-dev.1 (http://www.colorado.edu/physics/phet/dev/html/molecule-shapes/1.0.1-dev.1/molecule-shapes_en.html) with fullscreen support enabled by default (if available for the browser).

**Comment by jonathanolson** [comment]
Assign to @ariel-phet for priority.

**Comment by ariel-phet** [comment]
Closing this issue (since there is quite a bit of older info), and creating a new up-to-date task.
